lib/index.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
```

Comment:
Updating to latest ESLint made the `'use strict'` appear as warning, just to mitigate that.

Of course we had no benefit from using (if we follow standards) it and we lose nothing without.

Is just that "the future" is strict by default, I like that. :beers:

See also:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode
- http://eslint.org/docs/rules/#strict-mode